cmd/restic/integration_test.go

```diff
@@ -662,7 +662,7 @@ func TestBackupTags(t *testing.T) {
 		"expected no tags, got %v", newest.Tags)
 	parent := newest
 
-	opts.Tags = []string{"NL"}
+	opts.Tags = []restic.TagList{[]string{"NL"}}
```

Comment:
`opts.Tags = restic.TagLists{[]string{"NL"}}` would be a bit more precise, but ok :wink:

Reply:
Fixed 😉